### ISSUE TITLE
Comparision with Perkeep

### ISSUE THREAD

**Issue by AlekSi** [timelinize/timelinize#16]
I wonder how it compares to [Perkeep and other similar software](https://perkeep.org/doc/compare).

**Comment by AlekSi** [comment]
It would be nice to have it on the website (once there is a website)

**Comment by mholt** [comment]
Yeah, agreed. The website needs a lot of work, and a thorough explanation of the project's place in the world (esp. compared to other tools) will be very necessary.
